plugins/postgres/dbt/adapters/postgres/impl.py

```diff
@@ -1,9 +1,15 @@
 from datetime import datetime
 from dataclasses import dataclass
-from typing import Optional, Set, List, Any
+from typing import Any, Optional, Set, List
 
 from dbt.adapters.base.meta import available
-from dbt.adapters.base.impl import AdapterConfig, AdapterFeature, ConstraintSupport
+from dbt.adapters.base.impl import (
```

Comment:
We should pull these from its new location `dbt.adapters.capability`

Reply:
Done.